### ISSUE TITLE
Does this need better UI?

### ISSUE THREAD

**Issue by blazgocompany** [BlockliveScratch/Blocklive#19]
I like good UI... I can volunteer to make this popup UI better if you want?

**Comment by ScratchFakemon** [comment]
You should make a fork and make a pull request for your new UI

**Comment by blazgocompany** [comment]
no... I meant, good UI is not the most important thing (functionality is more important), so is this really wanted?

**Comment by ScratchFakemon** [comment]
Yeah Blocklive needs better UI and to... not... be... BROKEN

**Comment by MeechaPooch** [comment]
We're having hosting issues haha. Oracle automatically banned the instance Ive been hosting BL on for 3 years because i created a reverse ssh tunnel into a github codespace-- i think the open connection triggered some detector on OC

**Comment by MeechaPooch** [comment]
id love to see what you can do with the ui- I'd appreciate a dedicated settings page for sure. No promises about integrating your changes, but if ilike them i will!

**Comment by blazgocompany** [comment]
Ok, I am glad y'all agree. You can see some of my work in UI and animation at develop.rainai.pages.dev

**Comment by blazgocompany** [comment]
One more question, how do you test it?

**Comment by blazgocompany** [comment]
@MeechaPooch One more question, how do you test it?
